pkg/controller/replication_controller_test.go

```diff
@@ -320,9 +267,22 @@ func TestSynchonize(t *testing.T) {
 		}),
 		T: t,
 	}
+	fakeUpdateHandler1 := util.FakeHandler{
+		StatusCode:   200,
+		ResponseBody: runtime.EncodeOrDie(testapi.Codec(), &controllerSpec1),
+		T:            t,
+	}
+	fakeUpdateHandler2 := util.FakeHandler{
+		StatusCode:   200,
+		ResponseBody: runtime.EncodeOrDie(testapi.Codec(), &controllerSpec2),
+		T:            t,
+	}
+
 	mux := http.NewServeMux()
 	mux.Handle("/api/"+testapi.Version()+"/pods/", &fakePodHandler)
 	mux.Handle("/api/"+testapi.Version()+"/replicationControllers/", &fakeControllerHandler)
+	mux.Handle(fmt.Sprintf("/api/"+testapi.Version()+"/replicationControllers/%s", controllerSpec1.Name), &fakeUpdateHandler1)
+	mux.Handle(fmt.Sprintf("/api/"+testapi.Version()+"/replicationControllers/%s", controllerSpec2.Name), &fakeUpdateHandler2)
```

Comment:
Hm, I think I may have added it, but now that we've changed a bunch of stuff about our URL paths, that testapi.Version() call is not fooling anyone (only works for v1beta1/v1beta2). not important to change, just making a note.